[2022-03-20 12:53:48,119] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2022-01-18T00:00:00+00:00 [queued]>
[2022-03-20 12:53:48,137] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2022-01-18T00:00:00+00:00 [queued]>
[2022-03-20 12:53:48,138] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 12:53:48,138] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-03-20 12:53:48,138] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 12:53:48,180] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_user> on 2022-01-18T00:00:00+00:00
[2022-03-20 12:53:48,184] {standard_task_runner.py:52} INFO - Started process 66293 to run task
[2022-03-20 12:53:48,204] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'processing_user', '2022-01-18T00:00:00+00:00', '--job-id', '490', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmplko7na8z', '--error-file', '/tmp/tmpvfe0tiuq']
[2022-03-20 12:53:48,205] {standard_task_runner.py:77} INFO - Job 490: Subtask processing_user
[2022-03-20 12:53:48,322] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.processing_user 2022-01-18T00:00:00+00:00 [running]> on host airflowvm
[2022-03-20 12:53:48,518] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2022-01-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-18T00:00:00+00:00
[2022-03-20 12:53:48,588] {python.py:151} INFO - Done. Returned value was: None
[2022-03-20 12:53:48,602] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20220118T000000, start_date=20220320T125348, end_date=20220320T125348
[2022-03-20 12:53:48,727] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-20 12:53:48,748] {local_task_job.py:151} INFO - Task exited with return code 0
